rest_api/controller/documents.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from haystack.schema import Document
+from haystack.document_store import sql
+from rest_api.config import LOG_LEVEL
+from typing import Dict
+from fastapi import APIRouter, HTTPException
+import logging
+
+sql = sql.SQLDocumentStore()
+now = datetime.now()
+SUCCESS: int = 200
+CREATED: int = 201
```

Comment:
We can directly use [HTTPStatus](https://docs.python.org/3/library/http.html#http.HTTPStatus) enum.